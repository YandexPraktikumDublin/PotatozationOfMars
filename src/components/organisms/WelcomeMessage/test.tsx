import React from 'react'
import { mount } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { WelcomeMessage } from '.'

describe('<WelcomeMessage />', () => {
  it('should renders correct <WelcomeMessage />', () => {
    const wrapper = mount(<WelcomeMessage />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
