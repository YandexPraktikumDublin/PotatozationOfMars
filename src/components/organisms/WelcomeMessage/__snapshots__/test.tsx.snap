// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WelcomeMessage /> should renders correct <WelcomeMessage /> 1`] = `
<WelcomeMessage>
  <div
    className="fixed bottom-0 max-w-full"
    style={
      Object {
        "height": "30.625rem",
        "maxHeight": "calc(100vh - 4.5rem)",
        "right": "3.5625rem",
        "width": "44rem",
      }
    }
  >
    <div
      className="bg-contain bg-no-repeat text-center absolute top-0 z-10 flex px-14"
      style={
        Object {
          "backgroundImage": "url(function SvgBubble(props) {
  return React.createElement(
    'svg', 
    Object.assign({}, props, {'data-file-name': SvgBubble.name})
  );
})",
          "height": "18.75rem",
          "left": "0",
          "width": "25rem",
        }
      }
    >
      <div
        className="font-bold m-auto"
      >
        You think I launched Tesla into space as a joke? No way, it\`s a secret potatozation of Mars project. Click any button to help me.
      </div>
    </div>
    <img
      alt="Elon Musk"
      className="absolute bottom-0 z-auto h-full w-auto"
      src={[Function]}
      style={
        Object {
          "right": "0",
        }
      }
    />
  </div>
</WelcomeMessage>
`;
